### ISSUE TITLE
Animated GIF Images no longer animate after being resized

### ISSUE THREAD

**Issue by venyii** [VoycerAG/gridfs-image-server#7]
Source GIF: http://i.imgur.com/NDaqA6t.gif
Resized GIF: http://i.imgur.com/h8pagpP.gif

**Comment by sharpner** [comment]
thanks for the report. I will look into animated gifs.

**Comment by sharpner** [comment]
Which resize typ was used? Could you provide the configuration?

**Comment by venyii** [comment]
```json
{
    "allowedEntries": [
        {
            "name": "700x525f",
            "width": 700,
            "height": 525,
            "type": "fit"
        }
    ]
}

**Comment by sharpner** [comment]
this can be fixed after #10
